manifest.toml

```diff
@@ -47,6 +47,12 @@ ram.runtime = "100M"
     type = "path"
     default = "/gitea"
 
+    [install.indexer]
+    ask.en = "Enable code search"
+    help.en = "Indexing code can take a lot of system resources, especially when first enabled or on major Gitea upgrade"
+    type = "boolean"
+    default = false
```

Comment:
I'm not sure we need to request this question at the install time. Having in the config panel is enough from my point of view.